TS.fsx

```diff
@@ -465,17 +465,33 @@ let EmitStaticInterface flavor (i:Browser.Interface) =
     // Some types are static types with non-static members. For example, 
     // NodeFilter is a static method itself, however it has an "acceptNode" method
     // that expects the user to implement. 
-    let hasNonStaticMember = 
-        match i.Methods with
-        | Some ms -> ms.Methods |> Array.exists (fun m -> m.Static.IsNone)
-        | _ -> false
+    let hasNonStaticMember =
+        let hasNonStaticMethod =
+            match JsonItems.getAddedItemsByInterfaceName ItemKind.Method flavor i.Name with
+            | [||] ->
+                match i.Methods with
+                | Some ms -> ms.Methods |> Array.exists (fun m -> m.Static.IsNone)
```

Comment:
why don't you have to also check `m.Static.Value` like in the added methods case?